<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SLCalibrationDialog</class>
 <widget class="QDialog" name="SLCalibrationDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>890</width>
    <height>540</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>890</width>
    <height>540</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>890</width>
    <height>540</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>SL Calibration</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <widget class="SLVideoWidget" name="videoWidget" native="true">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>651</width>
     <height>471</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="snapButton">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>205</x>
     <y>500</y>
     <width>241</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Please wait...</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>10</y>
     <width>181</width>
     <height>281</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="calibrateButton">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>718</x>
     <y>430</y>
     <width>111</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Calibrate</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>330</y>
     <width>181</width>
     <height>29</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Size (mm):</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="checkerSizeBox"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>360</y>
     <width>181</width>
     <height>29</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Rows:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="checkerRowsBox"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>390</y>
     <width>181</width>
     <height>29</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Cols:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="checkerColsBox"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>480</y>
     <width>181</width>
     <height>29</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QPushButton" name="cancelButton">
      <property name="text">
       <string>&amp;Cancel</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="saveButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>&amp;Save</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SLVideoWidget</class>
   <extends>QWidget</extends>
   <header>SLVideoWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>SLCalibrationDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>703</x>
     <y>498</y>
    </hint>
    <hint type="destinationlabel">
     <x>628</x>
     <y>510</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
